ci/build-and-send-slack-notification.ps1

```diff
@@ -0,0 +1,77 @@
+# Send a Slack notification with a link to the build.
+
+# Download previously uploaded slack attachments that were generated for each testing step
+New-Item -ItemType Directory -Path "$PSScriptRoot/slack_attachments"
+& buildkite-agent artifact download "*slack_attachment_*.json" "$PSScriptRoot/slack_attachments"
+
+$attachments = @()
+$all_steps_passed = $true
+Get-ChildItem -Recurse "$PSScriptRoot/slack_attachments" -Filter *.json | Foreach-Object {
+    $attachment = Get-Content -Path $_.FullName | Out-String | ConvertFrom-Json
+    if ($attachment.color -eq "danger") {
+        all_steps_passed = $false
```

Comment:
```suggestion
        $all_steps_passed = $false
```
Can you also test purposefully with a failing test?

Reply:
Good catch! Tested it now: 
![image](https://user-images.githubusercontent.com/54983025/69262376-4dc88c00-0bbb-11ea-8fc5-332a45ead63c.png)